arkiverer/src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/fileservice/FilestorageService.kt

```diff
@@ -32,15 +36,16 @@ class FilestorageService(@Qualifier("basicWebClient") private val webClient: Web
 		.block()!!
 
 
-	override fun getFilesFromFilestorage(key: String, data: Soknadarkivschema): List<FilElementDto> {
+	override fun getFilesFromFilestorage(key: String, data: Soknadarkivschema): List<FileData> {
 		val timer = metrics.filestorageGetLatencyStart()
 		try {
 			val fileIds = getFileIds(data)
 			logger.info("$key: Getting files with ids: '$fileIds'")
 
 			val files = getFiles(fileIds)
+				.map { FileData(it.uuid, it.fil, OffsetDateTime.now(ZoneOffset.UTC)) }
```

Comment:
Soknadsfillager still returns `FilElementDto`s, but here we convert it to the new data type that will be used instead.